Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java

```diff
@@ -42,6 +49,32 @@ public void initDefaultCommand() {
 		setDefaultCommand(new TeleopDrive());
 	}
 
+	/**
+	 * Sets the left side of the drivetrain to use the talons' pids to run at the
+	 * specified speed.
+	 * 
+	 * @param value
+	 *            The speed to run at
+	 */
+	public void dtLeftPIDDrive(double value) {
+		double setValue = value * Robot.getConst("Units per 100ms", 3413);
+		dtLeftMaster.set(ControlMode.Velocity, setValue);
+		dtLeftSlave.set(ControlMode.Velocity, setValue);
```

Comment:
Shouldn't (isn't) slave set to follow master? If so, it shouldn't be trying to do it's own PID, right?